resources/views/user/show.blade.php

```diff
@@ -65,8 +65,13 @@
         @feature('discord')
         <tr>
           <th>Discord Username:</th>
-          <td>{{ $user->getProfile()->getDiscordUsername() }}</td>
-        </tr>
+            <td>
+              {{ $user->getProfile()->getDiscordUsername() }}
+              @if ($user->getProfile()->getDiscordUserSnowflake())
+                <i>({{ $user->getProfile()->getDiscordUserSnowflake() }})</i>
+              @endif
+            </td>
+          </tr>
```

Comment:
remove one level of indentation (2 spaces)

Reply:
the whole td block, not just the <i>

Reply:
and the trailing </tr>

Reply:
oh yeah, oops! sorry :grin: